private_sharing/models.py

```diff
@@ -74,6 +87,35 @@ def badge_upload_path(instance, filename):
     return 'direct-sharing/badges/{0}/{1}'.format(instance.id, filename)
 
 
+def project_membership_visible(user, project):
+    """
+    Determine if the user's membership in a project is visible or not.
+    """
+    project_id = id_label_to_project(project)
+    user_id = user_to_id(user)
+
+    if project_id is not None:
+        project = DataRequestProjectMember.objects.get(member_id=user_id,
```

Comment:
I think this resulted in an error when I tried this out locally, I think due to an obsolete badge. Since badges can be out of sync, maybe this should have a `try` / `except DataRequestProjectMember.DoesNotExist`?